Exiled.API/Features/Player.cs

```diff
@@ -128,6 +128,21 @@ public Player(GameObject gameObject)
         /// </summary>
         public static Dictionary<int, Player> IdsCache { get; } = new(20);
 
+        /// <summary>
+        /// Gets or sets a <see cref="Dictionary{TKey, TValue}"/> containing cached <see cref="RoleType"/> and their FF multiplier. This is for non-unique roles.
+        /// </summary>
+        public Dictionary<RoleType, int> FriendlyFireRules { get; set; } = new();
```

Comment:
Rename to `FriendlyFireMultiplier` or `FriendlyFireMultiplierValues`

Reply:
Sure